### DISCUSSION TITLE
FileNotFoundError when I migrate a new model using Flask-Migrate

### DISCUSSION THREAD

**Discussion by Olamidun** [miguelgrinberg/Flask-Migrate#473] [Q&A]
Hello,

I am working on an existing project that already had some models, I added my own model file and when I tried to run migrate using `python manage.py db migrate,` I got the error below:

`FileNotFoundError: [WinError 3] The system cannot find the path specified: 'C:\\Users\\user\\Desktop\\Work\\exampleproject\\example-app\\migrations\\versions'`

I know the error has to do with the facts that I do not have the versions folder in my project, how do I get it? what command do I run?

**Comment by miguelgrinberg** [comment]
See the [documentation](https://flask-migrate.readthedocs.io/en/latest/) for instructions on how to use this package. Also see my blog post [How to add Flask-Migrate to an existing project](https://blog.miguelgrinberg.com/post/how-to-add-flask-migrate-to-an-existing-project).
